src/api/c++/z3++.h

```diff
@@ -448,14 +506,23 @@ namespace z3 {
     public:
         param_descrs(context& c, Z3_param_descrs d): object(c), m_descrs(d) { Z3_param_descrs_inc_ref(c, d); }
         param_descrs(param_descrs const& o): object(o.ctx()), m_descrs(o.m_descrs) { Z3_param_descrs_inc_ref(ctx(), m_descrs); }
+        param_descrs(param_descrs && o) noexcept : object{std::forward<object>(o)}, m_descrs{o.m_descrs} {
```

Comment:
the object bit is redundant

Reply:
What do you mean? The word? That is just the template argument the standard library's `std::forward` requires, since we are only forwarding the `object` piece of the class.